### PR TITLE
Translation string updates

### PR DESCRIPTION
**Pull request by soulBit**
Add translation string requested here https://github.com/DistributedCollective/Sovryn-frontend/issues/143#issuecomment-771746234

Also added missing translation strings to the Spanish json file

### CODE REVIEW

**Review by creed-victor — APPROVED** [review]

**Comment by creed-victor** [comment]
@bitcoinuser most likely you will want to just suggest translation in this PR so I will wait for you before merging.

**Comment by bitcoinuser** [comment]
> @bitcoinuser most likely you will want to just suggest translation in this PR so I will wait for you before merging.

Hi, better you merge first so I can see the context to translate correctly. Thanks

File: src/locales/pt_br/translation.json [hunk 1]
```diff
@@ -664,7 +664,8 @@
       "positionEntryPrice": "Preço de entrada aproximado",
       "entryPriceNote": "O preço é fornecido pelo nosso AMM e, como tal, está sujeito a flutuações.",
       "tradeButtonLong": "Confirmar",
-      "tradeButtonShort": "Confirmar"
+      "tradeButtonShort": "Confirmar",
+      "closeBtn": "Close Dialog"
```
Comment (bitcoinuser) [inline src/locales/pt_br/translation.json:668]:
```suggestion
      "closeBtn": "Fechar janela"
```

**Review by bitcoinuser — CHANGES_REQUESTED** [review]
Translate to PT-BR

**Pull request merged**